src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -282,67 +281,67 @@
 "codeBase"="$PackageFolder$\Microsoft.VisualStudio.CoreUtility.dll"
```

Comment:
Okay, figured this out - looks like we're going to have to split this pkgdef in two, they were added to reduce the number of probing paths: https://github.com/dotnet/roslyn/issues/11146.